src/test/java/io/fabric8/vertx/maven/plugin/it/StartStopMojoIT.java

```diff
@@ -0,0 +1,302 @@
+package io.fabric8.vertx.maven.plugin.it;
+
+import com.google.common.collect.ImmutableMap;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.net.URL;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicReference;
+
+import static com.jayway.awaitility.Awaitility.await;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.fail;
+
+/**
+ * @author <a href="http://escoffier.me">Clement Escoffier</a>
+ */
+public class StartStopMojoIT {
+
+    private static String VERSION;
+    private static ImmutableMap<String, String> VARIABLES;
+
+    String ROOT_START = "projects/start-it";
+    String ROOT_OPTS = "projects/start-java-opts-it";
+    String ROOT_MAIN = "projects/start-with-main-class-it";
+    String ROOT_MAIN_EXPLODED = "projects/start-with-main-class-exploded-it";
+    String ROOT_EXPLODED = "projects/start-exploded-it";
+    String ROOT_WITH_CONF = "projects/start-with-conf-it";
+    String ROOT_WITH_CONF_EXPLODED = "projects/start-with-conf-exploded-it";
+    private Verifier verifier;
+
+
+    @BeforeClass
+    public static void init() {
+        File constants = new File("target/classes/vertx-maven-plugin.properties");
+        assertThat(constants.isFile());
+        Properties properties = new Properties();
+        try (FileInputStream fis = new FileInputStream(constants)) {
+            properties.load(fis);
+        } catch (IOException e) {
+            fail("Cannot load " + constants.getAbsolutePath(), e);
+        }
+
```

Comment:
Because, if you have cleaned the project and just run the test from you IDE, it should fail early. So there are case (outside of a pure maven build) where the file may not be there.